packages/playground/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@threefold/playground",
+  "version": "2.0.0-rc8",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "run-p type-check build-only",
+    "preview": "vite preview",
+    "test:unit": "vitest",
+    "build-only": "vite build",
+    "type-check": "vue-tsc --noEmit -p tsconfig.vitest.json --composite false",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
+    "format": "prettier --write src/"
+  },
+  "dependencies": {
+    "@mdi/font": "^7.2.96",
+    "@threefold/grid_client": "2.0.0-rc8",
+    "bip39": "^3.1.0",
+    "decimal.js": "^10.4.3",
+    "highlight.js": "^11.7.0",
+    "lodash": "^4.17.21",
+    "pinia": "^2.0.32",
+    "qrcode": "^1.5.1",
+    "tf_gridproxy_client": "^1.0.2",
+    "tfgrid-gql": "^1.0.1",
+    "validator": "^13.9.0",
+    "vue": "^3.2.47",
+    "vue-router": "^4.1.6",
+    "vuetify": "^3.1.15",
+    "web-ssh-keygen": "^0.1.2"
+  },
+  "devDependencies": {
+    "@rushstack/eslint-patch": "^1.2.0",
```

Comment:
please remove all the linting package from here, they should be on the mono repo root only